### PR TITLE
API to eject and remove ISO disk from VM

### PR DESCRIPTION
**Pull request by msftpgayam**

### CODE REVIEW

**Review by madhanrm — DISMISSED** [review]

File: rpc/common/moc_common_common.proto [hunk 1]
```diff
@@ -253,9 +253,10 @@ enum ProviderAccessOperation {
 	VirtualMachine_Start 				= 604;
 	VirtualMachine_Stop 				= 605;
 	VirtualMachine_Reset 				= 606;
-	VirtualMachine_Repair_Guest_Agent		= 607;
+	VirtualMachine_Repair_Guest_Agent  		= 607;
 	VirtualMachine_Pause 				= 608;
 	VirtualMachine_Save 				= 609;
+	VirtualMachine_Remove_Iso_Disk		= 610;
```
Comment (apluchik) [inline rpc/common/moc_common_common.proto:259]: nit: spacing, not sure if the build will adjust it

**Review by apluchik — APPROVED** [review]

File: rpc/common/moc_common_common.proto (hunk 1 above)
Comment (msftpgayam) [reply rpc/common/moc_common_common.proto]: Fixed

**Review by apluchik — APPROVED** [review]

**Review by rjaini — APPROVED** [review]

**Pull request merged**